lib/app/sequence_base.rb

```diff
@@ -944,9 +948,15 @@ def invalid_binding_message(invalid_binding, binding_def)
           code_as_string = "#{invalid_binding[:element].system}|#{invalid_binding[:element].code}"
         end
 
-        "#{invalid_binding[:resource].resourceType}/#{invalid_binding[:resource].id} " \
-        "at #{invalid_binding[:resource].resourceType}.#{binding_def[:path]} with code '#{code_as_string}' " \
-        "is not in #{binding_def[:system]}"
+        if binding_def[:system]
+          "#{invalid_binding[:resource].resourceType}/#{invalid_binding[:resource].id} " \
+          "at #{invalid_binding[:resource].resourceType}.#{binding_def[:path]} with code '#{code_as_string}' " \
+          "is not in #{binding_def[:system]}"
```

Comment:
How about something like this rather than repeating the entire message:
```
binding_entity = binding_def[:system].presence || 'the declared CodeSystem'
"#{invalid..." \
"..." \
"is not in #{binding_entity}"
```

Reply:
I like this way of doing it, yeah.